### DISCUSSION TITLE
QGIS wouldn't sync from QField, then QField replaced my current project with old cloud project

### DISCUSSION THREAD

**Discussion by mousehawk** [opengisch/QField#4236] [Bugs]
As per the title. I've just spent a day mapping areas at a site using my phone, pushed and synced several times through the day with no errors. Then tried to update the project on QGIS on my laptop and nothing changed. I tried to push and sync again from my phone, and my phone replaced the up-to-date local copy with the old version on the cloud. However there are some shapes on the "new" version that I added right at the end of the day so I don't understand WTF is going on.

Why isn't there a safeguard to stop the app replacing new local files with old cloud files? QFieldSync on my laptop asks whether to sync local > cloud or cloud > local - why doesn't the app?

Is there any way I can recover my work? The "Changes" page in app.qfield.cloud is just a massive list of errors and Not_applied. Nothing seems to change when I try Action > Apply.

Going forward, how do I sync stuff from my phone to my laptop?

**Comment by lukas-nietsch** [comment]
I have a quite simular issue right now. A colleageu mapped 2 days in a row without any upload or synch and today he accidantally pressed synchronize not upload data.
The Qfield App replaced the mapped data with the outdated data stored in the cloud project.
Is there a way to recover the mapped local data in QField App on the iPhone?

**Comment by herrsei123** [comment]
I have the same problem. After mapping a whole day i pressed uploading in QField. It says all done successfully. But when i  sync in QGIS it says it downloading new data but afterwards its no new data are present at all. 
Also it is dangerous that with cloud projects no manual backups are possible and the project data are stored hidden (e.g. outsite the ordinary data folder). so all is dependend on a well secure syncing app. but its way to easy to hit the wrong button. so it would be good to have an option for manually backup in my opinion or a solid protection of changed files. thank you.

Edit: Ok uploading worked correctly. So its an issue with the QGIS Plugin. Working with QGIS 3.22.12 and QField sync 4.4.9

**Comment by BigEyePuna** [comment]
Same here.  I can collect data on my android phone and use the "push changes" button with no errors. Then when I use the "synchronize Current Cloud Project" button in the QField Sync plugin on my machine in the office, none of those changes appear.  Why are there no responses to this issue?

**Reply by INTIAPRG** [reply]
HI! Have you solved the problem? I am having the same issue

**Reply by serverul** [reply]
Hi. Yes, it works well on android. On iOS I still have the same issue. I think that its a path issue. Solved on android.

**Reply by INTIAPRG** [reply]
I have the problem when using  Android...I can only see the changes in my desktop if I download the whole project each time, it doesn't work with teh synchronise button. I think is a path issue too, which local directory do you use? the user's cloud folder?

**Reply by serverul** [reply]
I use the local folder that qfield created in user directory. Doesn't work for me from other folder. Also I transformed all the layers în geopackage. I had partial sync when used shape files.

**Reply by INTIAPRG** [reply]
Nice! I will try, thanks :)

**Comment by mspoel80** [comment]
Same here. Also the upload from Qgis to Qfield Cloud does not go smootly. half of the time i get an failed status (no qgs file uploaded, but i can see the file in Cloud.

**Comment by serverul** [comment]
Same issue here. Some polygons edited on my iPhone appear in Qgis, but no points and most importantly no photos uploaded to qgis.  How can I resolve this issue? Thanks.

**Comment by camplinq** [comment]
Same issue here

**Comment by c04vu5** [comment]
This is why I hate cloud computing and sync
